File: src/main/java/com/matamercer/microblog/models/entities/File.java
package com.matamercer.microblog.models.entities;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Table(name = "files")
@Getter
@Setter
@NoArgsConstructor
public class File extends BaseModel{

    @Column(nullable = false)
    public String name;

    @ManyToOne
    @JoinColumn(name = "blog_id")
    private Blog owner;


}

File: src/main/java/com/matamercer/microblog/models/entities/File.kt
package com.matamercer.microblog.models.entities

import javax.persistence.*

@Entity
@Table(name = "files")
class File(
    @Column(nullable = false)
    var name: String,

    @ManyToOne
    @JoinColumn(name = "blog_id")
    var owner: Blog? = null,
    ) : BaseModel() {

}